tests/unit/lms/views/api/grading_test.py

```diff
@@ -33,7 +35,9 @@ def test_it_does_not_record_result_if_score_already_exists(
     def test_it_passes_the_callback_if_there_is_no_score(
         self, pyramid_request, lti_grading_service, LTIEvent
     ):
-        lti_grading_service.read_result.return_value = None
```

Comment:
This made the test still pass.

In light of this maybe a better design will be to optionally return this structure and then `falsy` tests like this would still be possible.

I'd like to explore this after releasing the bugfix.